shared/constants/urls.ts

```diff
@@ -0,0 +1,4 @@
+export enum BaseUrl {
+  Portfolio = 'https://app.metamask.io',
+  MetaMask = 'https://metamask.io',
+}
```

Comment:
can we still get rid of these @rekmarks

Reply:
The answer to this was no: `BaseUrl` is still used.